app/src/main/java/at/aau/edu/wizards/ui/gameboard/claim/GuessViewHolder.kt

```diff
@@ -1,17 +1,34 @@
 package at.aau.edu.wizards.ui.gameboard.claim
 
+import android.content.Context
+import android.os.VibrationEffect
+import android.os.Vibrator
+import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.RecyclerView
 import at.aau.edu.wizards.databinding.ItemGuessBinding
 
 class GuessViewHolder(
     private val binding: ItemGuessBinding,
     private val onClick: (Int) -> Unit,
+    private val context: Context
 ) : RecyclerView.ViewHolder(binding.root) {
 
     fun bind(value: Int) {
         binding.text.text = value.toString()
         binding.root.setOnClickListener {
             onClick(value)
+
+            // Trigger haptic feedback
+            val vibrator = ContextCompat.getSystemService(context, Vibrator::class.java)
```

Comment:
An improvement would be moving the vibrator to a private property of the view holder. If you want to make it fancy place it inside `by lazy` to avoid creation if not needed.